src/data.py

```diff
@@ -42,7 +42,12 @@ def _readODS(file):
 
 def _readCSV(file):
     try:
-        dt = pd.read_csv(file, encoding="iso-8859-1", skiprows=1, delimiter=";")
+        if "mpal" in file.casefold():
```

Comment:
Seria bom deixar documentado, pode ser no script mesmo como comentário, o porquê de pular 1 linha quando for "mpal". Assim melhora a compreensão dos demais desenvolvedores.